__tests__/app.test.js

```diff
@@ -234,3 +234,29 @@ describe('DELETE: /api/comments/:comment_id', () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("200: Responds with an object of all the users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users).toHaveLength(4);
+        body.users.forEach(({ username, name, avatar_url }) => {
+          expect(typeof username).toBe("string");
+          expect(typeof name).toBe("string");
+          expect(typeof avatar_url).toBe("string");
+        })
+      });
+  });
+  test('404: Responds with an error message if no users are found', () => {
```

Comment:
are we requesting to /api/users? we are making a request to an endpoint that doesn't exist on our server - we are getting a 404 route not found for the route rather than not getting back any users as we haven't requested any users!
So if you've already got this test before, then we don't need it and if you dont already have it then it should go into its own describe block
There isn't anything else we can test for, for the users endpoint!